view-admin/src/pages/prize-management.tsx

```diff
@@ -0,0 +1,1002 @@
+import type { NextPage } from "next";
+import Image from "next/image";
+import styles from "@/styles/prize-management.module.css";
+import { Header, Button } from "@/components/common";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+
+const Page: NextPage = () => {
+  const router = useRouter();
+  const [selectedNumbers, setSelectedNumbers] = useState<number[]>([]);
+  const toggleNumber = (number: number) => {
+    if (selectedNumbers.includes(number)) {
+      setSelectedNumbers(selectedNumbers.filter((n) => n !== number));
+    } else {
+      setSelectedNumbers([...selectedNumbers, number]);
+    }
+  };
+  useEffect(() => {
+    console.log("selectedNumbers:", selectedNumbers);
+  }, [selectedNumbers]);
+
+  const [searchText, setSearchText] = useState("");
+  const [searchDone, setSearchDone] = useState(false);
+
+  useEffect(() => {
+    if (searchDone) {
+      const elements = Array.from(document.querySelectorAll("p"));
+      elements.forEach((element) => {
+        if (
+          element &&
+          element.textContent &&
+          element.textContent.includes(searchText)
+        ) {
+          element.scrollIntoView({ behavior: "smooth", block: "center" });
+        }
+      });
+    }
+    setSearchDone(false);
+  }, [searchText, searchDone]);
+
+  const handleSearch = () => {
+    setSearchDone(true);
+  };
+
+  const [selectAll, setSelectAll] = useState<boolean>(false);
+  const toggleSelectAll = () => {
+    if (selectAll) {
+      setSelectedNumbers([]);
+    } else {
+      setSelectedNumbers([
+        1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
+        21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31,
+      ]);
+    }
+    setSelectAll(!selectAll);
+  };
+
+  return (
+    <div className={styles.container}>
+      <Header user="">
+        <div className={styles.main}>
+          <Button size="m" shape="circle" onClick={() => router.push("/")}>
+            <div className={styles.buttonContents}>
+              番号入力ページ
```

Comment:
```suggestion
              番号入力ページ
```
景品の方は「景品管理」なのでページつけるかつけないか統一した方が良いです。私的にページはいらない

Reply:
対応しました。